packages/components/src/internal/components/search/SampleFinderSection.tsx

```diff
@@ -151,8 +155,9 @@ export const SampleFinderSection: FC<Props> = memo(props => {
             newFilterCards.splice(index, 1);
 
             updateFilters(filterChangeCounter + 1, newFilterCards);
+            api.query.incrementClientSideMetricCount(SAMPLE_FILTER_METRIC_AREA, 'deleteFilter');
```

Comment:
I don't see this scenario (removing a filter card) called out in spec. It probably doesn't hurt to add, especially since we store the filters in session and it might be interesting to see if user actually prefer to start from clean state and have to manually delete previous filters.

Reply:
I would like to know if the saving the previous filters in the session is useful for folks or not (it's an open question for me whether we release with that functionality or not), but I don't think we'll be able to distinguish the removal on first coming to the page vs. removal at some other time from these metrics, so I'm not sure what decisions we'll make knowing how many times filters were deleted.

Reply:
OK. I can remove this metric.